tests/apps/courses/test_cms_wizards_blogpost.py

```diff
@@ -315,7 +315,7 @@ def test_cms_wizards_blogpost_submit_form_invalid_slug(self):
         self.assertFalse(form.is_valid())
         self.assertEqual(
             form.errors["slug"][0],
-            "Enter a valid 'slug' consisting of letters, numbers, underscores or hyphens.",
+            "Enter a valid “slug” consisting of letters, numbers, underscores or hyphens.",
```

Comment:
Why did you replace single quotes?

Reply:
this error message is in Django's code and they seem to have replaced the single quotes to be more literal...